legal-api/migrations/versions/5238dd8fb805_.py

```diff
@@ -16,58 +16,58 @@
 branch_labels = None
 depends_on = None
 
-role_enum = postgresql.ENUM('amalgamating', 'holding', 'primary', name='amalgamating_business_role')
-amalgamation_type_enum = postgresql.ENUM('regular',
+role_enum = postgresql.ENUM('amalgamating', 'holding', 'primary', name='amalgamating_businesses_role')
+amalgamations_type_enum = postgresql.ENUM('regular',
                                          'vertical',
                                          'horizontal',
-                                         name='amalgamation_type')
+                                         name='amalgamations_type')
 
 
 def upgrade():
 
     # add enum values
     role_enum.create(op.get_bind(), checkfirst=True)
-    amalgamation_type_enum.create(op.get_bind(), checkfirst=True)
+    amalgamations_type_enum.create(op.get_bind(), checkfirst=True)
 
     # ==========================================================================================
-    # amalgamating_business/amalgamation tables
+    # amalgamating_businesses/amalgamations tables
     # ==========================================================================================
 
     op.create_table(
-        'amalgamation',
+        'amalgamations',
         sa.Column('id', sa.Integer(), nullable=False),
         sa.Column('legal_entity_id', sa.Integer(), nullable=False),
         sa.Column('filing_id', sa.Integer(), nullable=False),
-        sa.Column('amalgamation_date', sa.TIMESTAMP(timezone=True), nullable=False),
+        sa.Column('amalgamations_date', sa.TIMESTAMP(timezone=True), nullable=False),
```

Comment:
I think some columns should remain singular instead of plural, e.g. `amalgamtion_date`, `amalgamation_id`.

Reply:
this change is not required